pkgs/desktops/deepin/deepin-wm/deepin-wm.plugins-dir.patch

```diff
@@ -0,0 +1,28 @@
+From ec15e3c6886f4bd8ab9e1da8cf3fac018da98008 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Jos=C3=A9=20Romildo=20Malaquias?= <malaquias@gmail.com>
+Date: Fri, 19 Apr 2019 23:56:33 -0300
+Subject: [PATCH] deepin.deepin-wm: set plugin dir from environment variable
+
+---
+ src/PluginManager.vala | 5 ++++-
+ 1 file changed, 4 insertions(+), 1 deletion(-)
+
+diff --git a/src/PluginManager.vala b/src/PluginManager.vala
+index 9ff8856..169bca1 100644
+--- a/src/PluginManager.vala
++++ b/src/PluginManager.vala
+@@ -58,7 +58,10 @@ namespace Gala
+ 				return;
+ 			}
+ 
+-			plugin_dir = File.new_for_path (Config.PLUGINDIR);
++                        string? plugin_dir_name = GLib.Environment.get_variable ("DEEPIN_WM_PLUGINS_DIR");
++                        if (plugin_dir_name == null || plugin_dir_name == "")
++                        	plugin_dir_name = Config.PLUGINDIR;
```

Comment:
Can you add `debug ();` messages?
So is `plugin_dir` set and its value or it's not and the value of `Config.PLUGINDIR`?

Reply:
> So is `plugin_dir` set and its value or it's not and the value of `Config.PLUGINDIR`?

Both are accepted, and the environment variable has priority.

Reply:
> Can you add `debug ();` messages?

Done.

Reply:
> Both are accepted, and the environment variable has priority.

Oh no, that's what I meant for the debug message to get across.